AutoYaST/SLES-12-SPn-to-SLES-15-SP5/README.md

```diff
@@ -0,0 +1,17 @@
+This autoyast file has been used to upgrade from SLES12SP5 to SLES15SP5.
+The SLES12 system has only the mandatory repositories (modules) activated.
+
+Please do not change the kernel option according to the SUMA documentation.
+Disabling the self update feature in combination with self_update=0 pt.options=self_update and using a distribution based on GM (first official release) media DVD, will delete the installation rather then upgrading it.
```

Comment:
Is this a bug in SUMA docs or GM image?

Reply:
The problem is with the GM media, if the selfupdate is disabled, then this problem can not be avoided.

Reply:
At the same time, it is a problem in the SUMA documentation:
Original text:
In the Kernel Options field, type any options to be passed to the kernel when booting for the installation. The install= parameter and the self_update=0 pt.options=self_update parameter are added by default.

Reality is:
There are two kernel parameters created during the creation of a distribution. 
install=[URL of installation media]
self_update=[URL of the self installer repository]

If the customer follows the SUMA documentation, then the self_updte reference will be changed, and this will disable the auto update feature.